dataland-internal-storage/src/test/resources/db/migration/V17/originalNonFinancial.json

```diff
@@ -0,0 +1,18 @@
+{
+  "general": {
+    "fiscalYearEnd": "2024-05-08",
+    "assurance": {
+      "value": "None",
+      "provider": null,
+      "dataSource": null
+    },
+    "referencedReports": {
+      "SustainabilityReport": {
+        "fileReference": "50a36c418baffd520bb92d84664f06f9732a21f4e2e5ecee6d9136f16e7e0b63",
+        "fileName": "SustainabilityReport",
+        "isGroupLevel": null,
+        "reportDate": null
+      }
```

Comment:
sollte der nicht ein currency field haben? oder zumindestens einen Report ergänzen der alle felder hat?

Reply:
this is for the (non-normal) case that such field didn't even exist in the first place. Its to make sure the migration doesn't have any weird behavior